src/open/clients/ElementData.js

```diff
@@ -0,0 +1,18 @@
+import {Maturity, Platform, FDroidLink, AppleStoreLink, PlayStoreLink, WebsiteLink, FlathubLink} from "../types.js";
+
+export const data = {
+    "id": "element.io",
+    "platforms": [Platform.Android, Platform.iOS, Platform.Windows, Platform.macOS, Platform.Linux, Platform.DesktopWeb],
+    "icon": "element.svg",
+    "appleAssociatedAppId": "7J4U792NQT.im.vector.app",
+    "name": "Element",
+    "description": "Fully-featured Matrix client, used by millions.",
+    "homepage": "https://element.io",
+    "author": "Element",
+    "maturity": Maturity.Stable,
+    "applestorelink": new AppleStoreLink('vector', 'id1083446067'),
+    "playstorelink": new PlayStoreLink('im.vector.app'),
+    "fdroidlink": new FDroidLink('im.vector.app'),
+    "flathublink": new FlathubLink('im.riot.Riot'),
```

Comment:
why not make these links something like this? You can wrap these strings into their *Link objects in  Client.getInstallLinks, that way it's closer to json and people don't have to deal with knowing about these classes:
```
"installLinks": {
  "applestore": ["vector", "id1083446067"],
  "playstore": "im.vector.app",
  "fdroid": "im.vector.app",
  "flathub": "im.riot.Riot",
  "website": "https://element.io/get-started"
}

Reply:
I would like as much of the content as possible to be generated by [squiddy](https://github.com/thibaultamartin/squiddy) (to make sure it will be kept up to date).

The file format for squiddy is not completely dry yet, but for clients it would look like

```toml
[[clients]]
title = "Element Web"
description = "Element is a glossy web client with an emphasis on performance and usability"
maturity = "Stable"
(more fields…)
appstore_details.org = "vector"
appstore_details.app_id = "id1083446067"
playstore_app_id = "im.vector.app"
fdroid_app_id = "im.vector.app"
flathub_app_id = "im.vector.app"
otherinstall_link = ["https://element.io/get-started"]
full_description = """
Longer multi-line
description of the client
that can [include links](#) and `markdown in general`
"""
[[clients.authors]]
name = "Element"
[[clients.authors]]
name = "Kat"
matrix_id = "@kittykat:matrix.org"
[[clients.authors]]
name = "Danielle Kirkwood"
matrix_id = "@daniellekirkwood:one.ems.host"
```

Under those circumstances, it's not a huge problem to use the classes. Is it acceptable for you?